File: src/Aicl.Colmetrik.Host.Web/WebApp/modules/app/model/Equipo.js
Ext.define('App.model.Equipo',{
	extend: 'Ext.data.Model',
	idProperty: 'Id',
	fields: 
	[
		{
			name: 'Id',
			type: 'int'
		},
		{
			name: 'Nombre',
			type: 'string'
		},
		{
			name: 'Fabricante',
			type: 'string'
		},
		{
			name: 'Modelo',
			type: 'string'
		},
		{
			name: 'Rangos',
			type: 'string'
		},
		{
			name: 'Resolucion',
			type: 'string'
		},
		{
			name: 'Especificacion',
			type: 'string'
		},
		{
			name: 'DispositivoIndicadorAjuste',
			type: 'string'
		},
		{
			name: 'Otros',
			type: 'string'
		},
		{
			name: 'Manual',
			type: 'string'
		}
	]
});

File: src/Aicl.Colmetrik.Model/Types/Equipo.cs
using System;
using System.ComponentModel.DataAnnotations;
using ServiceStack.Common;
using ServiceStack.DataAnnotations;
using ServiceStack.DesignPatterns.Model;

namespace Aicl.Colmetrik.Model.Types
{
	/// <summary>
	/// Description of Equipo.
	/// </summary>
	public partial class Equipo:IHasId<System.Int32>
	{
		public Equipo()
		{
		}
		
		[AutoIncrement]
		public int Id {get; set;}  
        
		[StringLength(100)]
		[Required]
        public string Nombre {get; set;} 
        
        [StringLength(100)]
        public string Fabricante {get; set;} 
        
        [StringLength(100)]
        public string Modelo {get; set;} 
        
        [StringLength(500)]
        public string Rangos {get; set;} 
		
        [StringLength(500)]
        public string Resolucion {get; set;} 
        
        [StringLength(510)]
        public string Especificacion {get; set;} 
        
        [StringLength(100)]
        public string DispositivoIndicadorAjuste {get; set;} 
        
        [StringLength(510)]
        public string Otros {get; set;} 
        
        [StringLength(510)]
        public string Manual {get; set;} 
	}
}
